tests/test_framework/mod.rs

```diff
@@ -452,6 +452,8 @@ pub fn verify_swap_results(
         assert!(
             balances.swap == Amount::from_btc(0.00442712).unwrap() // Successful coinswap
                 || balances.swap == Amount::from_btc(0.00442714).unwrap() // Recovery via timelock
+                || balances.swap == Amount::from_btc(0.00441884).unwrap() //Successful coinswap variant
+                || balances.swap == Amount::from_btc(0.00441886).unwrap() //Successful coinswap variant
```

Comment:
Are these extra asserts required? It just changed the previous assert value?

If its the second case, then those abort values should be replaced with the new value.